tests/test_config_entries.py

```diff
@@ -5729,6 +5729,14 @@ async def test_starting_config_flow_on_single_config_entry(
             None,
             {"type": data_entry_flow.FlowResultType.ABORT, "reason": "not_implemented"},
         ),
+        (
+            {"source": config_entries.SOURCE_ZEROCONF},
```

Comment:
We should also add a test of a user flow that is allowed when there's an ignored config entry already.

Reply:
That's covered by the existing test `test_starting_config_flow_on_single_config_entry_2`, with this parametrization:
```py
        (
            {"source": config_entries.SOURCE_USER},
            None,
            {"type": data_entry_flow.FlowResultType.ABORT, "reason": "not_implemented"},
        ),
```

Reply:
Ok.